backend/app/models/spree/backend_configuration.rb

```diff
@@ -62,61 +29,109 @@ def initialize(
     # end
     #
     # @!attribute menu_items
-    #   @return [Array<Spree::BackendConfiguration::MenuItem>]
+    #   @return [Array<Spree::MenuItem>]
     attr_writer :menu_items
 
     # Return the menu items which should be drawn in the menu
     #
     # @api public
-    # @return [Array<Spree::BackendConfiguration::MenuItem>]
+    # @return [Array<Spree::MenuItem>]
     def menu_items
       @menu_items ||= [
-        MenuItem.new(
+        Spree::MenuItem.new(
           ORDER_TABS,
           'shopping-cart',
           condition: -> { can?(:admin, Spree::Order) },
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           PRODUCT_TABS,
           'th-large',
           condition: -> { can?(:admin, Spree::Product) },
           partial: 'spree/admin/shared/product_sub_menu'
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           REPORT_TABS,
           'file',
           condition: -> { can?(:admin, :reports) },
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           CONFIGURATION_TABS,
           'wrench',
           condition: -> { can?(:admin, Spree::Store) },
           label: :settings,
           partial: 'spree/admin/shared/settings_sub_menu',
           url: :admin_stores_path
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           PROMOTION_TABS,
           'bullhorn',
           partial: 'spree/admin/shared/promotion_sub_menu',
           condition: -> { can?(:admin, Spree::Promotion) },
           url: :admin_promotions_path
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           STOCK_TABS,
           'cubes',
           condition: -> { can?(:admin, Spree::StockItem) },
           label: :stock,
           partial: 'spree/admin/shared/stock_sub_menu',
           url: :admin_stock_items_path
         ),
-        MenuItem.new(
+        Spree::MenuItem.new(
           USER_TABS,
           'user',
           condition: -> { Spree.user_class && can?(:admin, Spree.user_class) },
           url: :admin_users_path
         )
       ]
     end
+
+    # New quick switch items can be added to the admin:
+    #
+    # Spree::Backend::Config.configure do |config|
+    #   config.quick_switch_items << config.class::QuickSwitchItem.new(
+    #     [:o, :order],
+    #     :find_and_redirect_to_order,
+    #     "o ORDER_NUMBER"
+    #   )
+    # end
+    #
+    # @!attribute quick_switch_items
+    #   @return [Array<Spree::QuickSwitchItem>]
+    attr_writer :quick_switch_items
+
+    # Return the quick switch items that administrators can search by
+    #
+    # @api public
+    # @return [Array<Spree::QuickSwitchItem>]
+    def quick_switch_items
+      @quick_switch_items ||= [
+        Spree::QuickSwitchItem.new(
+          [:o, :order],
+          :find_and_redirect_to_order,
+          Spree.t("quick_switch.help_actions.order")
```

Comment:
Looks like these translations are not quite available when this code gets run. Does anyone know a way to get around this?